containers/Loan/Issue/index.tsx

```diff
@@ -16,9 +16,10 @@ interface Props {
 
```

Comment:
registry has also to be part of the props so it can be set through the query params from  the url. Please note that therefore you also have to adjust the issue page code. And parse the registry from the query in the `getInitialProps` function so that you can pass it as props to this component